.github/workflows/pr-realm-js.yml

```diff
@@ -0,0 +1,337 @@
+name: pr-realm-js
+on: pull_request
+
+jobs:
+  build-and-test:
+    name: Build and test for ${{ matrix.variant.os }} ${{ matrix.variant.arch }}
+    runs-on: ${{ matrix.variant.runner }}
+    env:
+      npm_config_realm_local_prebuilds: ${{github.workspace}}/prebuilds
+      REALM_DISABLE_SYNC_TESTS: true
+      REALM_DISABLE_ANALYTICS: 1
+    strategy:
+      #fail-fast: false
+      matrix:
+        variant:
+          - { os: linux, runner: ubuntu-latest, arch: x64, artifact-path: prebuilds, test-node: true, test-electron: true }
+          - { os: windows, runner: windows-2019, arch: x64, artifact-path: prebuilds, test-node: true, test-electron: true }
+          - { os: windows, runner: windows-2019, arch: ia32, artifact-path: prebuilds }
```

Comment:
Is there any specific reason why we're not using `windows-latest` here?

Reply:
After github's change to use windows 2022 (https://github.com/actions/virtual-environments/issues/4856) for runners, the windows builds were failing.